browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -178,9 +178,15 @@ EngineFlags ShouldBlockRequestOnTaskRunner(
     url_to_check = ctx->request_url;
   }
 
+  bool force_aggressive = SameDomainOrHost(
+      ctx->initiator_url,
+      url::Origin::CreateFromNormalizedTuple("https", "youtube.com", 80),
```

Comment:
should this be 443 instead of 80?

Reply:
This function only checks for the same domain or host, so the port doesn't really matter - it just needs to be present to create a `url::Origin`.